user-guide/settings/connections.md

```diff
@@ -20,7 +20,8 @@ __Fiddler listens on port:__ textbox controls which port Fiddler Everywhere uses
 
 ## Act as a System Proxy on Startup
```

Comment:
I guess Connections and Gateway sections are still to be updated with fewer images?

Reply:
Removed obsolete images and introduced the new styling with H4